### PR TITLE
Add support for File page in Content section of UI

### PR DESCRIPTION
**Pull request by sambible**
Adds Views/Entities for the Content -> Content Types -> File screen. 

Supporting this robottelo PR:

### CODE REVIEW

File: airgun/entities/file.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.utils import retry_navigation
+from airgun.views.file import FileDetailsView, FilesView
+
+
+class FilesEntity(BaseEntity):
```
Comment (vijaysawant) [inline airgun/entities/file.py:7]: Can you add doc string for classes and methods written in the entitites>file.py

**Review by vijaysawant — CHANGES_REQUESTED** [review]
Can you add doc string for classes and methods written in the `entitites>file.py` and `views>file.py`

File: airgun/entities/file.py (hunk 1 above)
Comment (sambible) [reply airgun/entities/file.py:7]: It's pretty much self explanatory, imo

**Review by vijaysawant — APPROVED** [review]

File: airgun/views/file.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    SatTab,
+    SatTable,
+)
+from airgun.widgets import Search
+
+
+class FilesView(BaseLoggedInView):
+    """Main Files view"""
+
+    title = Text("//h2[contains(., 'Files')]")
```
Comment (LadislavVasina1) [inline airgun/views/file.py:16]: As I am looking at the Files page the Title is h1 and not h2. Update the locator please `//h1[contains(.,'Files')]`

Comment (LadislavVasina1) [inline airgun/views/file.py:16]: As I am looking at the Files page the Title is h1 and not h2. Update the locator please.
```suggestion
    title = Text("//h1[contains(., 'Files')]")
```

**Review by LadislavVasina1 — CHANGES_REQUESTED** [review]
Apart from that one small change in a locator, it looks good to me, GJ!

**Review by damoore044 — APPROVED** [review]
ack pending locator comment!

**Review by LadislavVasina1 — APPROVED** [review]

**Pull request merged**